src/core/qfieldappauthrequesthandler.cpp

```diff
@@ -83,7 +85,15 @@ bool QFieldAppAuthRequestHandler::handleLayerLogins()
       }
       else
       {
-        emit reloadEverything();
+        const QList<QgsMapLayer *> mapLayers = QgsProject::instance()->mapLayers().values();
```

Comment:
Can we inject the `QgsProject` instead (as a property)?
It's already here: https://github.com/opengisch/QField/blob/master/src/core/qgismobileapp.cpp#L264 and the appauthrequesthandler is created in the same source file, so could easily be injected.